### DISCUSSION TITLE
Getting 404 Error Only On Mobile - Laptop Works Fine

### DISCUSSION THREAD

**Discussion by bethanybeachbum** [cotes2020/jekyll-theme-chirpy#1631] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Getting 404 error on mobile devices only.
Conversely, bethanybeachbum.github.io generated from "chirpy-starter' works fine on laptop
bethanybeachbum.github.io is public.

### Operations you have already tried

Restarted mobile phone and cleared cache to confirm 404 error.
Tried bethanybeachbum.github.io on second phone and still got error.
Tried both Safari and Chrome browser - still got 404 error.
All Github workflows are green.

### Anything else?

bethanybeachbum.github.io points to allenmarkbrown.com through Cloudflare.
Can't find anything in the forum referencing 404 Error on mobile only.

**Comment by kungfux** [comment, marked as answer]
JFYI: I cannot access https://allenmarkbrown.com/ from both mobile and desktop while https://bethanybeachbum.github.io/ is accessible. So, maybe it's not "mobile only" issue but problem with custom domain configuration.

**Reply by bethanybeachbum** [reply]
Thank you for your prompt reply.   You helped me realize that the custom domain option in the Settings -->Pages --> Custom Domain section of Github had lost my redirection to a custom domain. Appreciate your guidance.  What is scaring me is when I go back to that choice after saving it for a second time, it keeps dropping my custom domain name.
